Tangem/Modules/ExpressView/ExpressViewModel.swift

```diff
@@ -166,6 +181,100 @@ final class ExpressViewModel: ObservableObject {
     }
 }
 
+// MARK: - Provider slippage message
+
+extension ExpressViewModel {
```

Comment:
Хорошо бы тоже private 
```suggestion
private extension ExpressViewModel {
```